doc/using/configuration.chapter.md

```diff
@@ -100,14 +100,14 @@ There are several ways to tweak how Nix handles a package which has been marked
     }
     ```
 
-    For a more useful example, try the following. This configuration only allows unfree packages named roon-server and visual studio code:
+    The following example only allows two specific unfree packages:
 
     ```nix
     {
-      allowUnfreePredicate = pkg: builtins.elem (lib.getName pkg) [
-        "roon-server"
-        "vscode"
-      ];
+      allowUnfreePredicate = pkg: builtins.elem (lib.getName pkg) (map lib.getName [
+        roon-server
+        vscode-fhs
+      ]);
```

Comment:
I'm maybe overlooking something, but how do we access `vscode-fhs` in the `allowUnfreePredicate` before we've created a Nixpkgs instance?

Reply:
That `vscode-fhs` has to come from somewhere? AFAIU the snippet rn relies on a fixpoint, something like

```
let pkgs = import <nixpkgs>
  {
    config.allowUnfreePredicate = pkg: builtin.elem (pkgs.lib.getName pkg) (map pkg.lib.getName (with pkgs; [ roon-server vscode-fhs ]));
  }
in pkgs
```